clients/loader.py

```
@@ -0,0 +1,151 @@
+                system_prompt_additions=data.get('prompts', {}).get('system_prompt_additions', [])
+            )
+            
+            # Pre-compile regex patterns
+            for p in patterns:
+                try:
+                    config._compiled_patterns.append(re.compile(p, re.IGNORECASE))
+                except re.error as e:
+                    logger.warning(f"Invalid regex '{p}' in {path}: {e}")
+            
+            return config
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)